src/theory/strings/output_channel.h

```diff
@@ -0,0 +1,308 @@
+/*********************                                                        */
+/*! \file output_channel.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Andrew Reynolds
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2019 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Customized output channel for the theory of strings
+ **/
+
+#include "cvc4_private.h"
+
+#ifndef CVC4__THEORY__STRINGS__OUTPUT_CHANNEL_H
+#define CVC4__THEORY__STRINGS__OUTPUT_CHANNEL_H
+
+#include "context/cdhashset.h"
+#include "theory/uf/equality_engine.h"
```

Comment:
Add missing includes for `Context`, `Node`, `std::vector`, `OutputChannel`, `std::map`.